### PR TITLE
C++ String Tutorial & Demo

### PR DESCRIPTION
**Pull request by GirkovArpa**
Added C++ string tutorial & demo, using the Caesar cipher to demonstrate string manipulation.

### CODE REVIEW

**Comment by GirkovArpa** [comment]
How's this?

File: examples/strings/demo/cpp/caesar.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+typedef long int i32;
```
Comment (torch2424) [inline examples/strings/demo/cpp/caesar.cpp:1]: Can we rename the parent folder from `/cpp/` to `/c/` I forgot about that :upside_down_face: Thanks! :+1:

Comment (GirkovArpa) [reply examples/strings/demo/cpp/caesar.cpp]: Done!

**Comment by torch2424** [comment]
Perfect! Thank you very very much @GirkovArpa ! :smile: Merging this is now! :smile: :+1:

**Comment by torch2424** [comment]
Hmmmm, so it seems like my auto deploy script on my servers is acting up :thinking: @GirkovArpa I'll deploy this in a little bit :smile:

**Comment by torch2424** [comment]
Deployed! :smile: @GirkovArpa 

Screenshot from 2020-04-22 18-07-36

**Pull request merged**